src/components/ChannelPreview/__tests__/ChannelPreview.test.js

```diff
@@ -97,28 +98,38 @@ describe('ChannelPreview', () => {
       expect(getNodeText(getByTestId('unread-count'))).toBe('0');
     });
   });
-
-  it('should update the last event message & unreadCount, when message.new event is received', async () => {
-    const c = generateChannel();
-    await initializeChannel(c);
-
-    const { getByTestId } = render(getComponent());
-
-    await waitForElement(() => getByTestId('channel-id'));
-
-    const message = generateMessage({
-      user: clientUser,
-    });
-
-    channel.countUnread = () => 10;
-
-    act(() => {
-      dispatchMessageNewEvent(chatClient, message, channel);
-    });
-
-    await wait(() => {
-      expect(getNodeText(getByTestId('last-event-message'))).toBe(message.text);
-      expect(getNodeText(getByTestId('unread-count'))).toBe('10');
-    });
-  });
+  const eventCases = [
+    ['message.new', dispatchMessageNewEvent],
+    ['message.updated', dispatchMessageUpdatedEvent],
+    ['message.deleted', dispatchMessageDeletedEvent],
+  ];
+
+  it.each(eventCases)(
+    'should update the last event message & unreadCount on %s event',
```

Comment:
do you prefer this style vs the "`" one?

Reply:
does it make a difference?

Reply:
nope, just wondering for consistency across the codebase so we try to use the style everywhere

Reply:
lets keep `'`!! Same as react sdk